.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
Though I will say I feel like there's a fairly large distinction between arcs and canons, where canons should have pages that contain their entire thing, where as it seems silly generating that for every arc.